### PR TITLE
pkg: using environmental variable to decide which SAI to compile

### PR DESCRIPTION
**Pull request by stcheng**

### CODE REVIEW

**Comment by kcudnik** [comment]
Ack

File: debian/rules [hunk 1]
```diff
@@ -16,6 +16,7 @@ include /usr/share/dpkg/default.mk
 # package maintainers to append LDFLAGS
 #export DEB_LDFLAGS_MAINT_APPEND = -Wl,--as-needed
 
+DEB_SAIREDIS_FLAGS ?= "--with-sai=bcm"
```
Comment (lguohan) [inline debian/rules]: name to generic? 

how about DEB_SAI_FLAVOR ?= bcm

File: debian/rules [hunk 2]
```diff
@@ -28,7 +29,7 @@ include /usr/share/dpkg/default.mk
 #	-DCMAKE_LIBRARY_PATH=$(DEB_HOST_MULTIARCH)
 
 override_dh_auto_configure:
-	dh_auto_configure --
+	dh_auto_configure -- $(DEB_SAIREDIS_FLAGS)
```
Comment (lguohan) [inline debian/rules]: dh_auto_configure -- --with-sai=$(DEB_SAI_FLAVOR)

**Comment by stcheng** [comment]
updated

**Pull request merged**